internal/pkg/es/client.go

```diff
@@ -106,27 +92,3 @@ type InfoResponse struct {
 		Number string `json:"number"`
 	} `json:"version"`
 }
-
-func info(_ context.Context, es *elasticsearch.Client) (*InfoResponse, error) {
-	// Validate the connection
-	res, err := es.Info()
-
-	if err != nil {
-		return nil, err
-	}
-
-	defer res.Body.Close()
-
-	if res.IsError() {
-		return nil, fmt.Errorf("info fail %v", res)
-	}
-
-	var resp InfoResponse
```

Comment:
Is `InfoResponse` used anywhere else? can we get rid of it?

Reply:
Removed.